### PR TITLE
Implement socket to community as a group chat

### LINKED ISSUE #7: Conversation with another agent

### PR DESCRIPTION
**Pull request by ketphan02**
Closes #7 
Note:
- Need mongodb to store data
- Not  sure about the stability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ To launch the Elon Musk bot, first [install Python](https://realpython.com/insta
 pip install -r requirements.txt
 ```
 
+Then, start Docker and run:
+```
+docker pull mongo
+docker run mongo
+```
+
```
Comment (IvanIsCoding) [inline README.md:22]: If we used SQLite we would avoid having to run a Mongo instance

**Review by IvanIsCoding — COMMENTED** [review]
Thanks for coding the socket! My review is mostly about not using MongoDB as a dependency because I think we can use simpler things for a project of this size.

I suggest replacing the MongoDB with a SQLite database, which does not need a Docker instance running. Python has [SQLite support built in](https://docs.python.org/3/library/sqlite3.html), and we can probably leverage [SQLite JSON functions](https://devopsheaven.com/sqlite/databases/json/python/api/2017/10/11/sqlite-json-data-python.html) to achieve  a similar behavior without changing the code too much

File: README.md (hunk 1 above)
Comment (ketphan02) [reply README.md:22]: On it!

**Comment by ketphan02** [comment]
Don't merge this, this still has some problems when exiting the group chat

**Comment by ketphan02** [comment]
Everything is great now @IvanIsCoding, please approve and merge this!

**Comment by IvanIsCoding** [comment]
Closing as outdated code

**Pull request closed without merging**